test/unit/pending_blocks_test.exs

```diff
@@ -9,6 +9,8 @@ defmodule Unit.PendingBlocks do
   alias LambdaEthereumConsensus.Store.BlockStore
 
   setup do
+    Application.put_env(:lambda_ethereum_consensus, ChainSpec, config: MainnetConfig)
```

Comment:
why does this test fail with minimal config?

Reply:
Because it uses a block with mainnet-sized values